docs/guides.rst

```diff
@@ -277,33 +279,35 @@ When developing on a Mac or Linux host system, you may want to verify your websi
 
 1. **Get your Chassis machine's IP**
 
-   From the command line within your Chassis folder, run ``vagrant ssh -c 'ifconfig'``. This sends a message into your Chassis environment to output the VM's internal network information. What we're looking for is the IP address assigned to the VM, which in our example case looks like this: 
+   From the command line within your Chassis folder, run ``vagrant ssh -c 'ifconfig'``. This sends a message into your Chassis environment to output the VM's internal network information. What we're looking for is the IP address assigned to the VM, which in our example case looks like this:
+
+   ::
```

Comment:
FYI, this can just be

```
looks like this::
```

(That's where the double-colon syntax originally comes from)